src/Microsoft.TargetingPack.Private.CoreCLR/ref/Configurations.props

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <BuildConfigurations>
+      netcoreapp;
```

Comment:
I'm grabbing the CoreLib for UAP from Microsoft.TargetingPack.Private.NETNative which is why I made this project not to build on uap vertical